rules/windows/process_creation/proc_creation_win_susp_non_exe_image.yml

```diff
@@ -1,12 +1,14 @@
 title: Execution of Suspicious File Type Extension
 id: c09dad97-1c78-4f71-b127-7edb2b8e491a
 status: experimental
-description: Checks whether the image specified in a process creation event doesn't refer to an .exe file (caused by process ghosting or other unorthodox methods to start a process)
+description: |
+    Detects whether the image specified in a process creation event doesn't refer to an ".exe" (or other known executable extension) file. This can caused by process ghosting or other unorthodox methods to start a process.
+    This rule might require some initial baselining to align with some third party tooling in the user environment
```

Comment:
```suggestion
    Detects whether the image specified in a process creation event doesn't refer to an ".exe" (or other known executable extension) file. This can be caused by process ghosting or other unorthodox methods to start a process.
    This rule might require some initial baselining to align with some third party tooling in the user environment.
```